src/components/GoodsList/GoodsList.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const GoodList = (props) => {
+  const {
+    goods, goodsSelected, selectValueNumber, reverseButton, sortButton,
+    resetButton, sortByLength, selectButton,
+  } = props;
+
+  return (
+    <>
+      <button
+        type="button"
+        onClick={reverseButton}
+      >
+      Reverse
+      </button>
+
+      <button
+        type="button"
+        onClick={sortButton}
```

Comment:
Как понять как оно сортирует по названию `SortButton`?
```suggestion
        onClick={handleSortAlphabet}
```

переименуй также название методов в классе